src/vue/backend/src/components/APIKey.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+    <idx-form-group class="idx-wp-needs-validation {'was-validated': success|error}" novalidate>
```

Comment:
Prefer to use `customClass` here instead, also using the bitwise operator instead of the OR that you want `success || error`

Reply:
needs-validation {'was-validated': success|error} is there to show the VCL styles but likely won't be the final implementation, as this class should be applied to the form.